config.m4

```diff
@@ -217,9 +227,9 @@ if test "$PHP_MONGODB" != "no"; then
   "
 
   PHP_ARG_WITH(libbson, whether to use system libbson,
-  [  --with-libbson             Use system libbson], no, no)
+  [  --with-libbson          MongoDB: Use system libbson], no, no)
   PHP_ARG_WITH(libmongoc, whether to use system libmongoc,
-  [  --with-libmongoc           Use system libmongoc], no, no)
+  [  --with-libmongoc        MongoDB: Use system libmongoc], no, no)
```

Comment:
Missing `AC_HELP_STRING` for these and `mongodb-sasl` below. I'll take care of that myself.